examples/billboard/billboard.py

```diff
@@ -0,0 +1,187 @@
+# Location is taken from the billing export table provided by the user.
+def create_dataset(args):
+
+    #To be created dataset for billboard
+    dataset_id = "{}.{}".format(args.PROJECT_ID,args.BILLBOARD_DATASET_NAME_TO_BE_CREATED)
+    
+    #Exported billing dataset
+    source_id = "{}.{}.{}".format(args.PROJECT_ID,args.STANDARD_BILLING_EXPORT_DATASET_NAME,args.standard_table)
+   
+    #check if billboard dataset exists
+    if check_billboard_dataset_exists(dataset_id) == True:
+        return
+
+    # since we need to create, construct a full Dataset object to send to the API.
+    dataset = bigquery.Dataset(dataset_id)
+
+    #Need to create billboard dataset in the same location as exported dataset
+    table_info = bq_client.get_table(source_id)
+    dataset.location = table_info.location
+    print("Exported BQ Table GEO Location={} so creating billboard dataset will be created in there as well.".format(table_info.location))
+ 
+    
+    # Send the dataset to the API for creation, with an explicit timeout.
+    # Raises google.api_core.exceptions.Conflict if the Dataset already
+    # exists within the project.
+    dataset = bq_client.create_dataset(dataset, timeout=30)  # Make an API request.
+    print("Created dataset {}".format(dataset_id))
+
+
+# Creates the view for the Billboard
+def create_billboard_view(args,isStandard):
+
+    global output_url
+
+    if isStandard == True:
+        source_id="{}.{}.{}".format(args.PROJECT_ID,args.STANDARD_BILLING_EXPORT_DATASET_NAME,args.standard_table)
+        view_id="{}.{}.{}".format(args.PROJECT_ID,args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_standard)
+    else:
+        source_id="{}.{}.{}".format(args.PROJECT_ID,args.DETAILED_BILLING_EXPORT_DATASET_NAME,args.detailed_table)
+        view_id="{}.{}.{}".format(args.PROJECT_ID,args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_detailed)
+  
+    #Check resource detailed export is present
+    try:      
+        bq_client.get_table(source_id)
+    except NotFound:
+        if isStandard == True:
+            print("Standard usage cost export not found="+source_id +" so skipping billboard view creation")
+            sys.exit()
+        else:
+            print("Detailed usage cost export not found="+source_id +" so skipping billboard detailed view creation")
+        return;
+
+    sql = """
+    CREATE VIEW if not exists `{}`
+    AS select *, COALESCE((SELECT SUM(x.amount) FROM UNNEST(s.credits) x),0) AS credits_sum_amount, COALESCE((SELECT SUM(x.amount) FROM UNNEST(s.credits) x),0) + cost as net_cost, EXTRACT(DATE FROM _PARTITIONTIME) AS date from `{}` s WHERE _PARTITIONTIME >'2020-08-01'
+    """.format(
+        view_id,source_id
+    )
+
+    bq_view_client = bigquery.Client(project=args.PROJECT_ID) #not sure why this need project_id
+
+    job = bq_view_client.query(sql)  # API request.
+    job.result()  # Waits for the query to finish.
+
+    if isStandard == True:
```

Comment:
Redundant, `isStandard` is already boolean.

Please change to `if isStandard: `